src/olympia/devhub/templates/devhub/includes/addon_details.html

```diff
@@ -63,7 +63,7 @@
 </li>
 
 </li>
-{% if addon.current_version %}
+{% if addon.is_listed and addon.current_version %}
```

Comment:
Just to avoid the exception I'm raising temporarily in `current_version`, yes. Once I remove it and `current_version` returns `None` if there is no listed version, then the block would simply be hidden in that case.